### PR TITLE
examples: Add better helper method for adding tools dropdown

### PR DESCRIPTION
**Pull request by wayfarer3130**
### Context

The bindings for tools have a lot of steps required to get everything working, and those steps are repeated many times for different types of bindings.  This PR adds registration and an example set of tools for annotation tools to make it easier to create new tools.

### Testing

yarn example stackAnnotationTools
yarn example labelmapSegmentationDynamicThreshold

#### PR

- [] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [] My code has been well-documented (function documentation, inline comments,
  etc.)

- [] I have run the `yarn build:update-api` to update the API documentation, and have
  committed the changes to this PR. (Read more here https://www.cornerstonejs.org/docs/contribute/update-api)

#### Public Documentation Updates

- [] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [] "OS: 
- [] "Node version: 
- [] "Browser:
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

File: packages/tools/examples/labelmapSegmentationDynamicThreshold/index.ts [hunk 1]
```diff
@@ -190,17 +129,103 @@ thresholdOptions.set('CT Bone: (200, 1000)', {
...
 
-addDropdownToToolbar({
-  options: {
-    values: Array.from(thresholdOptions.keys()),
-    defaultValue: defaultThresholdOption,
+toolMap.set('ThresholdSphere', {
+  baseTool: BrushTool.toolName,
+  configuration: {
+    ...configuration,
+    activeStrategy: 'THRESHOLD_INSIDE_SPHERE',
```
Comment (sedghi) [inline packages/tools/examples/labelmapSegmentationDynamicThreshold/index.ts:153]: not yet right?

File: packages/tools/examples/stackAnnotationTools/index.ts [hunk 2]
```diff
@@ -209,62 +181,10 @@ async function run() {
...
-  cornerstoneTools.addTool(AngleTool);
-  cornerstoneTools.addTool(CobbAngleTool);
-  cornerstoneTools.addTool(ArrowAnnotateTool);
-  cornerstoneTools.addTool(PlanarFreehandROITool);
-  cornerstoneTools.addTool(EraserTool);
-  cornerstoneTools.addTool(KeyImageTool);
-
   // Define a tool group, which defines how mouse events map to tool commands for
   // Any viewport using the group
   const toolGroup = ToolGroupManager.createToolGroup(toolGroupId);
```
Comment (sedghi) [inline packages/tools/examples/stackAnnotationTools/index.ts:186]: I believe this example serves as a great starting point for new people joining the project. While advanced examples can be abstracted with these new utiltiies, keeping this one straightforward with the verbose API can help newcomers see how things work. Let's maintain this simplicity for basic examples like this and only refactor the advacned ones. Can you revert these chanages?

**Review by sedghi — COMMENTED** [review]
see my comment please

File: packages/tools/examples/stackAnnotationTools/index.ts (hunk 2 above)
Comment (wayfarer3130) [reply packages/tools/examples/stackAnnotationTools/index.ts:186]: Reverted as requested.  Moved the example to videoTools

File: packages/core/src/RenderingEngine/Viewport.ts [hunk 3]
```diff
@@ -232,8 +232,8 @@ class Viewport implements IViewport {
    * works with vtk-js backed rendering pipeline.
    *
    * @param flipOptions - Flip options specifying the axis of flip
-   * @param flipOptions.flipHorizontal - Flip the viewport on horizontal axis
-   * @param flipOptions.flipVertical - Flip the viewport on vertical axis
+   *    * flipOptions.flipHorizontal - Flip the viewport on horizontal axis
```
Comment (wayfarer3130) [inline packages/core/src/RenderingEngine/Viewport.ts:235]: These are just fixes so eslint doesn't complain.

**Review by sedghi — APPROVED** [review]

**Pull request merged**